data/genesis/demo-pos-base.toml

```diff
@@ -1,6 +1,7 @@
 base_version = "0.3"
 upgrade_version = "0.3"
-epoch_height = 300
+epoch_height = 100
+epoch_start_block = 1
```

Comment:
so I think we also need to change default value for `epoch_start_block`to 1 if its not provided in genesis.toml?

Reply:
Do you mean in the sequencer, or prover or both?

Currently it will be set to `0` at

https://github.com/EspressoSystems/espresso-network/blob/02bb5190ad9c772821d4777eb9471844c1d49540/sequencer/src/lib.rs#L372

this seems kind of wrong doesn't it?

Like if we used `main` code for `mainnet` right now it would be set to `0` there.

The hotshot network config does require a value to be set.

Reply:
yeah, we need to set it to 1 instead of 0 IMO

Reply:
I don't understand why 1 is okay. If we run the current binary with the current mainnet genesis we would be setting it to 1 but we're not configured to run with epochs or upgrade to it.

Reply:
So if we set this to 1 it breaks the prover for the demo native fee version.